File: accudrop/app/src/main/java/me/chrislane/accudrop/viewmodel/JumpViewModel.java
package me.chrislane.accudrop.viewmodel;

import android.arch.lifecycle.LiveData;
import android.arch.lifecycle.MutableLiveData;
import android.arch.lifecycle.ViewModel;


public class JumpViewModel extends ViewModel {

    private MutableLiveData<Integer> jumpId = new MutableLiveData<>();

    public LiveData<Integer> getJumpId() {
        return jumpId;
    }

    public void setJumpId(int jumpId) {
        this.jumpId.setValue(jumpId);
    }
}

File: app/src/main/java/me/chrislane/accudrop/viewmodel/JumpViewModel.kt
package me.chrislane.accudrop.viewmodel

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel


class JumpViewModel : ViewModel() {

    private val jumpId = MutableLiveData<Int>()

    fun getJumpId(): LiveData<Int> {
        return jumpId
    }

    fun setJumpId(jumpId: Int) {
        this.jumpId.value = jumpId
    }
}
